fakelibusb_test.go

```diff
@@ -108,6 +112,21 @@ func (f *fakeLibusb) getDevices(*libusbContext) ([]*libusbDevice, error) {
 	return ret, nil
 }
 
+func (f *fakeLibusb) wrapSysDevice(ctx *libusbContext, systemDeviceHandle uintptr) (*libusbDevHandle, error) {
+	if int(systemDeviceHandle) > len(fakeDevices)-1 {
```

Comment:
I think if you access an index that is not available as a system device, it should return an error instead